Aztec/Classes/TextKit/LayoutManager.swift

```diff
@@ -27,6 +27,9 @@ class LayoutManager: NSLayoutManager {
     ///
     var blockquoteBorderWidth: CGFloat = 2
 
+    /// The list indent style
+    ///
+    var listIndentStyle: TextList.IndentStyle = .default
```

Comment:
Nit: Since this only applies to unordered lists, we should consider making this clearer.